libs/sdk-ui-kit/src/Dialog/HubspotConversionTouchPointDialog.tsx

```diff
@@ -14,11 +14,17 @@ import {
 export const HubspotConversionTouchPointDialog: React.FC<IHubspotConversionTouchPointDialogBaseProps> = (
     props,
 ) => {
+    const onDialogSubmit = () => {
+        const submitBtn = document.querySelector(".hs-button.primary.large") as HTMLInputElement;
+        submitBtn?.click();
+    };
+
     return (
         <Dialog
             displayCloseButton={true}
             onCancel={props.onClose}
-            submitOnEnterKey={false}
+            submitOnEnterKey={true}
```

Comment:
Just a confirmation: is it true that we won't support submit on Tab key, as mentioned in the `Expected result` of https://jira.intgdc.com/browse/SD-1882?